queries.sql

```diff
@@ -117,4 +117,62 @@ WHERE O.full_name = 'Dean Winchester' AND escape_attempts = 0;
 SELECT COUNT(*) as count, full_name FROM animals A
 INNER JOIN owners O ON A.owner_id = O.id
 GROUP BY O.full_name
-ORDER BY count DESC;
\ No newline at end of file
+ORDER BY count DESC;
+
+
+--Who was the last animal seen by William Tatcher?
```

Comment:
@SelmaNdapanda, as the previous reviewer requested. You need to add LICENSE to your project. Also, make sure it's working when we click on the link. When I click on the link it's leading me to this [link](https://github.com/SelmaNdapanda/Vet-Clinic-Database/blob/add_join_table/LICENSE) _(broken link)_

Reply:
Sorry about that, The link is now working.